ReactViewControl/FrameInfo.cs

```diff
@@ -5,6 +5,8 @@ namespace ReactViewControl {
 
     internal class FrameInfo : IFrame {
 
+        internal const string MainViewFrameName = "";
```

Comment:
string.Empty

Reply:
string.empty is a static.. this is a const

Reply:
Oh, didn't notice it